pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "chord-drs"
-version = "0.17.1"
+version = "0.18.0"
```

Comment:
I think this should be a major version? Since this is a breaking change.

Reply:
since it's 0.x, minor versions can break the API: https://semver.org/#spec-item-4

Reply:
Ah didn't know that, nice.